docs/sections/workflows/writing_basics.md

```diff
@@ -175,103 +154,445 @@ See the {ref}`corresponding topics section for more details <topics:processes:us
+Then, there are other two instructions in `multiply()`:
+the first stores the `product` into the *context* of the work chain using `self.ctx` such that it can be used in the next step of the outline; and the second declares one of the outputs.
 
-```{literalinclude} include/code/multiply_add.py
+When we use the `self.out()`, the first argument (`'product'`) specifies the label of the output, which corresponds to the label provided to the spec in the `define()` method.
+The second argument is the result of the calc function.
+Note that we can only output Nodes, and `product` is one because it is the output of the calcfunction `multiplication`.
+
+The next step is to declare the `add()` method, which corresponds to the second step in the outline:
+
+```{code-block} python
+from aiida.orm import Int
+from aiida.engine import WorkChain, calcfunction
+
+@calcfunction
+def multiplication(x, y):
+    return x * y
+
+@calcfunction
+def add(x, y):
+    return x + y
+
+
+class MultiplyAddWorkChain(WorkChain):
+    """WorkChain to multiply two numbers and add a third, for testing and demonstration purposes."""
+
+    @classmethod
+    def define(cls, spec):
+        """Specify inputs and outputs."""
+        super().define(spec)
+
+        spec.input("x", valid_type=Int)
+        spec.input("y", valid_type=Int)
+        spec.input("z", valid_type=Int)
+
+        spec.outline(
+            cls.multiply,
+            cls.add
+        )
+
+        spec.output("product", valid_type=Int)
+        spec.output("final_result", valid_type=Int)
+
+    def multiply(self):
+        """Multiply two integers."""
+
+        product = multiplication(self.inputs.x, self.inputs.y)
+
+        # Passing to context to be used by other functions
+        self.ctx.product = product
+
+        # Declaring one of the outputs
+        self.out("product", product)
+
+
+    def add(self):
+        """Add two integers."""
+
+        # Call `add` using a variable from the context and one of the inputs
+        result = add( self.ctx.product, self.inputs.z )
+
+        # Parsing the output
+        self.out("final_result", result)
+```
+
+In `add()`, we called the `addition()` calculation function (declared outside of the class) using a variable from the context and one of the inputs.
+Then, we declared the last output of the work chain.
+And that is it.
+We finished our very first and simple work chain.
+
+Let's run it!
+In the terminal, navigate to the folder where you have your work chain file.
+Then, open a `verdi shell` session and execute:
+
+
+```{code-block} ipython
+In [1]: from aiida.engine import run
+In [2]: from multiply_add_workchain import MultiplyAddWorkChain
+In [3]: result = run(MultiplyAddWorkChain, Int(2), Int(3), Int(5) )
+In [4]: result
+Out[2]:
+{'product': <Int: uuid: 9c5c29de-6176-41fe-a051-672a5348e631 (pk: 1909) value: 6>,
+ 'final_result': <Int: uuid: 3dbebeb3-7ed4-4331-976c-94099a305622 (pk: 1911) value: 11>}
+```
+Check the list with last processes:
```

Comment:
Maybe I'm overly repeating things, but I especially think reminding them of the `-p` option is a good idea:

```suggestion
Check the list of processes with `verdi process list`, using `-a / --all` to also see _terminated_ processes and `-p / --past-days 1` to only see processes _created_ in the past day:
```